brew/vars.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+set -eu
+
+# TODO(mkramlich): commit hash/pin (for brew formula)
+# TODO(mkramlich): elim redundancy with or otherwise maybe take advantage of version/version.sh
+
+cat <<EOF
+TS_VER=1.5.0
+TS_BIN=/usr/local/bin
+TS_SOCK=/usr/local/var/run/tailscale/tailscaled.sock
+TS_STATE=/usr/local/var/lib/tailscale/tailscaled.state
+TS_LOG_DIR=/usr/local/var/log/tailscale
+TS_CELLAR=/usr/local/Cellar/tailscale
```

Comment:
This is not portable to M1 macs, their version of Homebrew uses /opt/homebrew/Cellar/tailscale

Reply:
thanks! ditto on my other platform permutations comment. I do think having "homebrew" in their paths is better in the longterm. /usr/local anything is getting cluttered/fuzzy